test/NuGet.Core.FuncTests/NuGet.Commands.FuncTest/RestoreCommandTests.cs

```diff
@@ -136,7 +138,7 @@ public async Task RestoreCommand_CannotFindProjectReferenceWithDifferentNameCase
             // Arrange
             var sources = new List<PackageSource>
             {
-                new PackageSource("https://www.nuget.org/api/v2/")
+                new PackageSource("https://api.nuget.org/v3/index.json")
```

Comment:
private const string on the class, so if it changes in the future (v4 api?) it only changes in 1 place?

Reply:
I'll look into that. I think we might already have some sort of a const/static.

Reply:
Found it https://github.com/NuGet/NuGet.Client/blob/dev/src/NuGet.Core/NuGet.Configuration/PackageSource/NuGetConstants.cs. 
Will use those.